### DISCUSSION TITLE
rtl433 not showing any sensor data (Accurite 515)

### DISCUSSION THREAD

**Discussion by marcvl64** [merbanan/rtl_433#2145] [General]
I'm not sure that this is the right forum. If not, I apologize in advance.

I'm running rtl_433 in a docker container on a RPi4 CM4 (with BigTreeTech Pad 5 touchscreen expansion board). Everything seems to be running fine, no errors. rtl_433 connects to the USB device and tunes to the right frequency. However, no sensor data is showing up. I know the Acurite sensors work; I know the USB stick works (tried it on a different RPi4). I'm baffled and not sure how to troubleshoot this further.

This is what the logs show:

> admin@raspberrypi:~/Containers/rtl433tomqtt $ docker-compose up rtl433tomqtt
> Starting rtl433tomqtt ... done
> Attaching to rtl433tomqtt
> rtl433tomqtt    | rtl_433 version unknown inputs file rtl_tcp RTL-SDR
> rtl433tomqtt    | Use -h for usage help and see https://triq.org/ for documentation.
> rtl433tomqtt    | Trying conf file at "rtl_433.conf"...
> rtl433tomqtt    | Trying conf file at "/home/user/.config/rtl_433/rtl_433.conf"...
> rtl433tomqtt    | Reading conf from "/home/user/.config/rtl_433/rtl_433.conf".
> rtl433tomqtt    | Publishing MQTT data to mosquitto port 1883
> rtl433tomqtt    | Publishing device info to MQTT topic "rtl_433/e466a871c32a/devices[/type][/model][/subtype][/channel][/id]".
> rtl433tomqtt    | Publishing events info to MQTT topic "rtl_433/e466a871c32a/events".
> rtl433tomqtt    | Publishing states info to MQTT topic "rtl_433/e466a871c32a/states".
> rtl433tomqtt    | Registered 145 out of 175 device decoding protocols [ 1-4 8 11-12 15-17 19-21 23 25-26 29-36 38-60 63 67-71 73-100 102-105 108-116 119 121 124-128 130-149 151-161 163-168 170-175 ]
> rtl433tomqtt    | Detached kernel driver
> rtl433tomqtt    | Found Rafael Micro R820T tuner
> rtl433tomqtt    | Exact sample rate is: 250000.000414 Hz
> rtl433tomqtt    | [R82XX] PLL not locked!
> rtl433tomqtt    | Sample rate set to 250000 S/s.
> rtl433tomqtt    | Tuner gain set to Auto.
> rtl433tomqtt    | Tuned to 433.920MHz.
> rtl433tomqtt    | Allocating 15 zero-copy buffers
> rtl433tomqtt    | MQTT Connected...
> rtl433tomqtt    | MQTT Connection established.
> rtl433tomqtt    | {"time" : "2022-08-10 21:54:30.259248", "enabled" : 145, "since" : "2022-08-10T21:44:30", "frames" : {"count" : 31, "fsk" : 33, "events" : 0}, "stats" : []}

Any ideas on what might be happening/how to troubleshoot or fix?

Marc

**Comment by zuckschwerdt** [comment]
Be sure not to plug the receiver directly into a USB port, avoid noise and use a (short) usb cable always.
On weak hardware use -Y squelch

Use `-M level -M noise` to see what the levels are.
Use `-Y autolevel` for better sensitivity.

Use `-vv` to show failed decoding attempts,
Use `-vvv` to show all decoding attempts,

To get a feel what rtl_433 is "seeing" try using the web api with `-F http`, then opening the URL, enable Modes->Raw(all).
Enable the pass-through output with `-F rtl_tcp` and connect with a UI (SigDigger, CubicSDR, Gqrx, …) with added https://github.com/pothosware/SoapyRTLTCP to see the live waterfall of the spectrum.

You can try https://triq.org/rtl_433/ANALYZE.html to capture a transmission.

**Reply by marcvl64** [reply]
Thanks. This helps a ton. I used the option -vvv and the acurite data is coming in but is not processed. These are the log entries (there are many more with list of all pulses but I truncated for now):

> rtl433tomqtt    | Badly formatted OS v2.1 message: {114} 05 56 ab 6a a8 15 5a ad aa a0 55 6a b6 aa 80 
> rtl433tomqtt    | Unrecognized Msg in OS v3: {114} 05 56 ab 6a a8 15 5a ad aa a0 55 6a b6 aa 80 
> rtl433tomqtt    | calibeur_rf104_callback: DECODE_FAIL_SANITY data all 0x00 or 0xFF
> rtl433tomqtt    | Exact bit width (in us) is 610.29 vs 500.00 (pulse width 227.43 vs 248.00), 7 bit preamble
> rtl433tomqtt    | Exact bit width (in us) is 610.86 vs 500.00 (pulse width 227.43 vs 248.00), 7 bit preamble
> rtl433tomqtt    | Exact bit width (in us) is 610.86 vs 500.00 (pulse width 227.43 vs 248.00), 7 bit preamble
> rtl433tomqtt    | Possible DSC Contact: bitbuffer:: Number of rows: 1 
> rtl433tomqtt    | [00] {12} c0 00     : 11000000 0000
> rtl433tomqtt    | row 0 bit count 12
> rtl433tomqtt    | DSC row 0 invalid bit count 12
> rtl433tomqtt    | acurite_txr_decode: bitbuffer:: Number of rows: 3 
> rtl433tomqtt    | [00] {50} c0 7a 48 11 ac 3f 00 : 11000000 01111010 01001000 00010001 10101100 00111111 00
> rtl433tomqtt    | [01] {50} c0 7a 48 11 ac 3f 00 : 11000000 01111010 01001000 00010001 10101100 00111111 00
> rtl433tomqtt    | [02] {50} c0 7a 48 11 ac 3f 00 : 11000000 01111010 01001000 00010001 10101100 00111111 00
> rtl433tomqtt    | acurite_txr_decode: row 0 bits 50, bytes 7 
> rtl433tomqtt    | acurite_txr_decode: skipping wrong len
> rtl433tomqtt    | acurite_txr_decode: row 1 bits 50, bytes 7 
> rtl433tomqtt    | acurite_txr_decode: skipping wrong len
> rtl433tomqtt    | acurite_txr_decode: row 2 bits 50, bytes 7 
> rtl433tomqtt    | acurite_txr_decode: skipping wrong len
> rtl433tomqtt    | acurite_986_decode: row 0 bits 161, bytes 5 
> rtl433tomqtt    | acurite_986_decode: skipping wrong len
> rtl433tomqtt    | radiohead_ask_extract: preamble not found
> rtl433tomqtt    | acurite_00275rm_decode: bitbuffer:: Number of rows: 3 
> rtl433tomqtt    | [00] {50} c0 7a 48 11 ac 3f 00 : 11000000 01111010 01001000 00010001 10101100 00111111 00
> rtl433tomqtt    | [01] {50} c0 7a 48 11 ac 3f 00 : 11000000 01111010 01001000 00010001 10101100 00111111 00
> rtl433tomqtt    | [02] {50} c0 7a 48 11 ac 3f 00 : 11000000 01111010 01001000 00010001 10101100 00111111 00
> rtl433tomqtt    | philips_aj3650_decode: wrong number of bits (162)
> rtl433tomqtt    | radiohead_ask_extract: preamble not found
> rtl433tomqtt    | AmbientWeather-TX8300: wrong size (161 bits)
> rtl433tomqtt    | philips_aj7010_decode: wrong number of bits (0)
> rtl433tomqtt    | gt_tmbbq05_decode: Possible Quigg BBQ: bitbuffer:: Number of rows: 1 
> rtl433tomqtt    | [00] {161} 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
> rtl433tomqtt    | norgo_decode: wrong size of bit per row 1
> rtl433tomqtt    | Exact bit width (in us) is 1226.00 vs 1072.00 (pulse width 631.00 vs 536.00), 4 bit preamble
> rtl433tomqtt    | Exact bit width (in us) is 1217.00 vs 1072.00 (pulse width 623.00 vs 536.00), 4 bit preamble
> rtl433tomqtt    | Exact bit width (in us) is 1219.00 vs 1072.00 (pulse width 624.00 vs 536.00), 4 bit preamble
> rtl433tomqtt    | cotech_36_7959_decode: Couldn't find preamble
> rtl433tomqtt    | secplus_v2_callback: DECODE_FAIL_SANITY
>

**Comment by marcvl64** [comment]
It seems to me that this is an issue topic, and not a discussion topic. I'll repost it under issues.
